src/controllers/PlantController.ts

```diff
@@ -2,7 +2,7 @@ import { Request, Response, NextFunction } from 'express';
 import PlantService from '../services/PlantService';
 
 class PlantController {
-  public service: PlantService = new PlantService();
+  constructor(private service: PlantService) {}
```

Comment:
Tenho percebido pelas mentorias que as pessoas estudantes entendem melhor o processo de criação de atributos quando eles são feitos fora do construtor (uma forma mais verbosa), acredito que ajude no entendimento.

```suggestion
  private readonly service: PlantService;  
  constructor(private service: PlantService) {
    this.service = service;
  }
```